docs/reference/hacl/hash/md5.md

```diff
@@ -1,14 +1,3 @@
 # MD5
 
 **Caution**: MD5 is insecure. Please avoid.
-
-## API Reference
```

Comment:
If there's nothing in here, it shouldn't get it's own page.

Reply:
Can we just remove MD5 from the documentation entirely?

Reply:
Okay, so I will delete the MD5 page and mention on the main hash page that MD5 is implemented but should not be used.